static/styles/zulip.css

```diff
@@ -1172,7 +1167,6 @@ div.message_table {
 .message_row {
     position: relative;
     border-left: 1px solid #e2e2e2;
```

Comment:
OK -- but do your changes create a duplicate left border?  Switching from `border-top` + `border-bottom` to just `border` certainly should set a left border...